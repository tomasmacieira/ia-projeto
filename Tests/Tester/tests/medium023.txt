ROW	1	3	3	1	3	1	4	0	4	0
COLUMN	0	1	4	0	1	4	2	2	1	5
2
HINT	2	5	C
HINT	2	9	M
