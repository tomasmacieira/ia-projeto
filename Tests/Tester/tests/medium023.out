.........t
..c....t.m
.....C.b.M
.........b
..t..lr...
..m.......
..b.lmr...
..........
.c...c..lr
..........
